File: BECore/app/model/ProjectAcceptanceBasic_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectAcceptanceBasic_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectAcceptanceBasic_AttachDetail', {
    extend: 'Ext.data.Model',

    idProperty: 'ProjectAcceptanceBasic_AttachDetailID',
    idType: 'Guid',

    fields: [
        { name: "ProjectAcceptanceBasic_AttachDetailID", type: "string" },
		{ name: "ProjectAcceptanceBasicID", type: "string" },
		{ name: "Contents", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
    ]
});

File: MT.Model/Business/ProjectAcceptanceBasic_AttachDetail.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
	#region ProjectAcceptanceBasic_AttachDetail:EntityBase
	/// <summary>
	/// This object represents the properties and methods of a ProjectAcceptanceBasic_AttachDetail:EntityBase.
	/// </summary>
	public class ProjectAcceptanceBasic_AttachDetail:EntityBase
	{
        [Key]
		public Guid ProjectAcceptanceBasic_AttachDetailID {get;set;}
		public Guid ProjectAcceptanceBasicID{get;set;}
		public string Contents{get;set;}
		public string FileName{get;set;}
		public string FileType{get;set;}
		public double FileSize{get;set;}
		public bool Inactive{get;set;}
		public int SortOrder{get;set;}

        public Guid? FileResourceID { get; set; }

    }
	#endregion
}

